### ISSUE TITLE
docker-coffea-dask

### ISSUE THREAD

**Issue by nsmith-** [CoffeaTeam/docker-coffea-base#7]
Do we want to make a new repo in this team named `docker-coffea-dask` and set up the same auto-update dependency, chained off of base? Or put it in this repo and make the CI rebuild all at once? What should be added? Here is what I had previously: https://github.com/CoffeaTeam/coffea-farm/blob/master/docker/coffea-dask/Dockerfile#L14-L28 
We'll need to add condor python deps both for casa and what @lgray was doing for LPC

**Comment by lgray** [comment]
Which condor python dependencies are you talking about, the bindings?

**Comment by nsmith-** [comment]
Yeah, e.g. `dask[dataframe] distributed dask-jobqueue` (do we need htcondor python at all?) etc.

**Comment by lgray** [comment]
We also need a somewhat patched version of `dask-jobqueue` for anything to work.
It doesn't use the python bindings as it is at present, and it can tweaked to work at fnal even without them.

So we can get away without `htcondor` python bindings unless we want to do something fancy.

**Comment by nsmith-** [comment]
casa needs some patches to distributed anyway so I think that's fine, certainly for now
https://github.com/CoffeaTeam/coffea-casa/blob/master/docker/coffea-casa/Dockerfile#L101-L105

**Comment by oshadura** [comment]
@nsmith- @lgray, I think I have a plan, please take a look:

since  coffea-casa is still in very `dev` mode, so I would prefer to have a stable `docker-coffea-dask` image similar to what you have in https://github.com/CoffeaTeam/coffea-farm/blob/master/docker/coffea-dask but in sync with `docker-coffea-base` and with a hook to detect a new tag of `dask` and `distributed` (they are always the same) so we can have always freshly tagged image? I am doing `dask` updates manually and it is sometimes a real 'nightmare'. For this it will make sense to have in a separate repo, I can do it quickly, just let me know if you agree. 

Then for coffea-casa, I will do 2 different images: 

1. one based on `docker-coffea-dask`, for Dask Scheduler with all tokens, dask-jobqueue, JupyterHub (we based this image on https://registry.hub.docker.com/r/jupyter/minimal-notebook so I will need to redo dependencies), HTCondor with config files that should be customizable (in my TODO list), `servicex-client`  and `servicex` config file generation.
2. other as well based on  `docker-coffea-dask`,  but used for Dask worker: with all tokens, dask-jobqueue,  HTCondor with config files and Dask worker setup.

All settings will be customizable and nothing hardcoded, so it could be easily reused on LPC or SSL.

Then later, if there will be a request we can add an image, something 'between'  `docker-coffea-dask` and `coffea-casa` images - a `docker-coffea-distrubuted` : the one with simple dask-jobqueue and HTCondor with config files.

**Comment by oshadura** [comment]
I will think If I can merge coffea-casa(1) with coffea-casa(2) images...

**Comment by oshadura** [comment]
Ah and I forgot about `coffea-spark` image :)

**Comment by nsmith-** [comment]
Sounds like a separate repo is our preference? I can create that (will just copy this one essentially)

**Comment by oshadura** [comment]
I can open PR in a new repository, I am very excited since I never tried to build an image derived from other image I build (e.g. `FROM coffeateam/coffea-base:latest`)
